calicoctl/commands/node/status.go

```diff
@@ -129,7 +136,19 @@ func drawTable(birdOut string) {
 
 		if s != "" {
 			col := []string{}
-			fields := strings.Fields(line)[3:6]
+			f := strings.Fields(line)[3:]
```

Comment:
`copy` is necessary because when you do `fields := f[0:3]` in Go they share the same underlying array, so if you modify one of those slices, the other one gets modified too! 

`f` is just a temp slice to hold the Fields (this is only when there is some connection error, since in that case there are more `fields` than what the table can hold) 

It is explained in the comment in line 143-145